tsconfig.app.json

```diff
@@ -4,7 +4,7 @@
     "outDir": "./dist/out-tsc",
     "types": []
   },
-  "files": ["src/main.ts"],
+  "files": ["src/main.ts", "src/polyfills.ts", "src/scope-polyfill/polyfill.ts", "src/scope-polyfill/css-style-sheet-updater.ts", "src/scope-polyfill/data.ts", "src/scope-polyfill/utils.ts"],
```

Comment:
Only the toplevel files are needed. The others are added automatically when they are imported.